hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemFileStatus.java

```diff
@@ -83,9 +85,11 @@ private FileStatus validateStatus(final AzureBlobFileSystem fs, final Path name,
       if (isDir) {
         assertEquals(errorInStatus + ": permission",
                 new FsPermission(DEFAULT_DIR_PERMISSION_VALUE), fileStatus.getPermission());
+        assertTrue(fileStatus.isDirectory());
```

Comment:
we always need error messages on simple asserttrue/false. Think to yourself "if this test failed and I had was the test report, what would I want to know". 

here: the filestatus. Something like
```
assertTrue("not a directory " + fileStatus, fileStatus.isDirectory())
```